markdown/appendices/resources.md

```diff
@@ -35,17 +53,22 @@ title: Other Doomsday Resources
 
 ## Videos
 
+- [LeFly runs a League with Meandeck (February 2021) - German](https://www.youtube.com/watch?v=pkpixIywm1w)
+- [Andreas Mengucci runs a league with UB Meandeck (December 2020)](https://www.youtube.com/watch?v=8HlIvDf7iDg)
+- [Anuraag Das runs some ANT/Doomsday with Ki Sawatari (December 2020)](https://www.youtube.com/watch?v=u-kyFx2VJ0c)
 - [Rodrigo *Sharkcaster_Mage* Togores runs a set of leagues with BUrg Meandeck (September 2020)]
   (https://www.youtube.com/playlist?list=PLowHzeuy7CRM8N34LB8Oc0XGCLIxB3Q3z)
-- [TwinlessTwinMTG runs a league with Monkeyscantcry]
+- [TwinlessTwinMTG runs a league with Monkeyscantcry (Expired)]
```

Comment:
Do we want to keep those? Since they're not available anymore

Reply:
Probably not. Which is a shame because they were good ¬.¬
Bloody people not saving vods properly.